src/test/java/com/conveyal/gtfs/GTFSTest.java

```diff
@@ -456,10 +481,43 @@ private void assertThatImportedGtfsMeetsExpectations(
                     LOG.info("Database record satisfies expectations.");
                     foundRecord = true;
                     break;
+                } else {
+                    LOG.error("Persistence mismatch on record {}", numRecordsSearched);
                 }
             }
             assertThatPersistenceExpectationRecordWasFound(numRecordsSearched, foundRecord, fieldsWithMismatches);
         }
+        // Check that error expectations match errors stored in database.
+        if (errorExpectations != null && errorExpectations.length > 0) {
```

Comment:
I think it'd be a good idea to also make sure there were zero errors reported if errorExpectations is null. That way we can be sure that clean feeds don't have errors.